src/libAtomVM/externalterm.h

```diff
@@ -64,7 +64,7 @@ term externalterm_to_term(const void *external_term, Context *ctx, int use_heap_
  * @returns the term deserialized from the input term, or an invalid term, if
  * deserialization fails.
  */
-enum ExternalTermResult externalterm_from_binary(Context *ctx, term *dst, term binary);
+enum ExternalTermResult externalterm_from_binary(Context *ctx, term *dst, term binary, size_t *bytes_read, size_t num_extra_terms);
```

Comment:
I think we should document the argument that we added here, it took me a while to understand the purpose of num_extra_terms.